### PR TITLE
Bump react-popper dependency to fix rare infinite loop

### LINKED ISSUE #2029: Tooltip2/Popover2 cause infinite loop when Chrome is zoomed out to or past 75%
### Bug report

- __Package version(s)__: v1.x and v2.x
- __Browser and OS versions__: Chrome 63, MacOS X 10.13.

#### Steps to reproduce

1. Zoom Chrome out to at least 75%
2. Open a Tooltip2 or Popover2

This is reproducible on the [blueprint v2 docs](http://blueprintjs.com/docs/v2/#core/components/tooltip) with the "Hover and click me" button when you click it.

A quick "fix" is to disable zooming on Chrome by setting `document.body.style.zoom` to `1 / window.devicePixelRatio` initially and every resize.

#### Actual behavior

Chrome freezes at 100% CPU usage with no console error.

This error is very inconsistent and appears to depend on the size of the popover/tooltip. I have a feeling this is related to Popper.js. Perhaps [this issue](https://github.com/FezVrasta/popper.js/issues/496)?

### PR DESCRIPTION
**Pull request by mfedderly**
#### Changes proposed in this pull request:

Incorporate https://github.com/popperjs/react-popper/pull/320

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2808,6 +2808,18 @@ deep-equal@^1.0.1:
   resolved "https://registry.yarnpkg.com/deep-equal/-/deep-equal-1.0.1.tgz#f5d260292b660e084eff4cdbc9f08ad3247448b5"
   integrity sha1-9dJgKStmDghO/0zbyfCK0yR0SLU=
 
+deep-equal@^1.1.1:
+  version "1.1.1"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/deep-equal/-/deep-equal-1.1.1.tgz#b5c98c942ceffaf7cb051e24e1434a25a2e6076a"
```
Comment (schlosna) [inline yarn.lock]: artifact url should be public https://registry.yarnpkg.com

File: yarn.lock [hunk 2]
```diff
@@ -5077,6 +5089,11 @@ is-alphanumerical@^1.0.0:
     is-alphabetical "^1.0.0"
     is-decimal "^1.0.0"
 
+is-arguments@^1.0.4:
+  version "1.0.4"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/is-arguments/-/is-arguments-1.0.4.tgz#3faf966c7cba0ff437fb31f6250082fcf0448cf3"
```
Comment (schlosna) [inline yarn.lock]: artifact url should be public https://registry.yarnpkg.com

File: yarn.lock [hunk 3]
```diff
@@ -2405,13 +2405,13 @@ create-react-class@^15.6.0:
     loose-envify "^1.3.1"
     object-assign "^4.1.1"
 
-create-react-context@<=0.2.2:
-  version "0.2.2"
-  resolved "https://registry.yarnpkg.com/create-react-context/-/create-react-context-0.2.2.tgz#9836542f9aaa22868cd7d4a6f82667df38019dca"
-  integrity sha512-KkpaLARMhsTsgp0d2NA/R94F/eDLbhXERdIq3LvX2biCAXcDvHYoOqHfWCHf1+OLj+HKBotLG3KqaOOf+C1C+A==
+create-react-context@^0.3.0:
+  version "0.3.0"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/create-react-context/-/create-react-context-0.3.0.tgz#546dede9dc422def0d3fc2fe03afe0bc0f4f7d8c"
```
Comment (schlosna) [inline yarn.lock]: artifact url should be public https://registry.yarnpkg.com

File: yarn.lock [hunk 4]
```diff
@@ -8466,13 +8483,14 @@ react-lifecycles-compat@^3.0.4:
   resolved "https://registry.yarnpkg.com/react-lifecycles-compat/-/react-lifecycles-compat-3.0.4.tgz#4f1a273afdfc8f3488a8c516bfda78f872352362"
   integrity sha512-fBASbA6LnOU9dOU2eW7aQ8xmYBSXUIWr+UmF9b1efZBazGNO+rcXT/icdKnYm2pTwcRylVUYwW7H1PHfLekVzA==
 
-react-popper@^1.3.3:
-  version "1.3.3"
-  resolved "https://registry.yarnpkg.com/react-popper/-/react-popper-1.3.3.tgz#2c6cef7515a991256b4f0536cd4bdcb58a7b6af6"
-  integrity sha512-ynMZBPkXONPc5K4P5yFWgZx5JGAUIP3pGGLNs58cfAPgK67olx7fmLp+AdpZ0+GoQ+ieFDa/z4cdV6u7sioH6w==
+react-popper@^1.3.7:
+  version "1.3.7"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/react-popper/-/react-popper-1.3.7.tgz#f6a3471362ef1f0d10a4963673789de1baca2324"
```
Comment (schlosna) [inline yarn.lock]: artifact url should be public https://registry.yarnpkg.com

File: yarn.lock [hunk 5]
```diff
@@ -8705,6 +8723,13 @@ regex-not@^1.0.0, regex-not@^1.0.2:
     extend-shallow "^3.0.2"
     safe-regex "^1.1.0"
 
+regexp.prototype.flags@^1.2.0:
+  version "1.2.0"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/regexp.prototype.flags/-/regexp.prototype.flags-1.2.0.tgz#6b30724e306a27833eeb171b66ac8890ba37e41c"
```
Comment (schlosna) [inline yarn.lock]: artifact url should be public https://registry.yarnpkg.com

**Comment by mfedderly** [comment]
Fixed by https://github.com/palantir/blueprint/pull/3885

**Pull request closed without merging**